packages/web-runtime/src/App.vue

```diff
@@ -24,7 +24,7 @@
             v-touch:swipe.left="handleNavSwipe"
             class="oc-app-navigation"
             :logo-img="logoImage"
-            :product-name="productName"
+            product-name="ffff"
```

Comment:
Was this meant to make it into the commit? ;)

Reply:
done 💯